pkgs/development/python-modules/ffmpeg-python/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ffmpeg
+, future
+}:
+
+buildPythonPackage rec {
+  pname = "ffmpeg-python";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    sha256 = "09s19kvgc4f0g05z3xjj8w15xcwbabmv3j0i1vppii978srms8k5";
+    inherit pname version;
+  };
+
```

Comment:
```suggestion

  nativeBuildInputs = [ pytest-runner ];
```
to avoid:
```
builder for '/nix/store/lr2xs105hj9q0f6hsa1n5q9bzr83v9p5-python3.8-ffmpeg-python-0.2.0.drv' failed with exit code 1; last 10 log lines:
      dist = best[req.key] = env.best_match(
    File "/nix/store/793ld8znz8abpkiwryyzdqhcgd6pqzfv-python3.8-setuptools-41.6.0/lib/python3.8/site-packages/pkg_resources/__init__.py", line 1065, in best_match
      return self.obtain(req, installer)
    File "/nix/store/793ld8znz8abpkiwryyzdqhcgd6pqzfv-python3.8-setuptools-41.6.0/lib/python3.8/site-packages/pkg_resources/__init__.py", line 1077, in obtain
      return installer(requirement)
    File "/nix/store/793ld8znz8abpkiwryyzdqhcgd6pqzfv-python3.8-setuptools-41.6.0/lib/python3.8/site-packages/setuptools/dist.py", line 787, in fetch_build_egg
      return cmd.easy_install(req)
    File "/nix/store/793ld8znz8abpkiwryyzdqhcgd6pqzfv-python3.8-setuptools-41.6.0/lib/python3.8/site-packages/setuptools/command/easy_install.py", line 673, in easy_install
      raise DistutilsError(msg)
  distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('pytest-runner')
```